build.gradle

```diff
@@ -398,8 +398,6 @@ tasks.withType(JavaCompile) {
 }
 
 tasks.test.finalizedBy(jacocoTestReport)  // report is always generated after tests run
-tasks.jacocoTestReport.dependsOn(test) // tests are required to run before generating the report
```

Comment:
This long could be causing the slowdown because all tasks are finalized by `jacocoTestReport` and `jacocoTestReport` depends on `test`